oid-info.md

```diff
@@ -0,0 +1,14 @@
+# Rekor OID Information
+
+## Description
+
+This document defines Rekor
+[OID values](https://github.com/sigstore/sigstore/blob/main/docs/oid-info.md).
+
+Rekor reserves the `1.3.6.1.4.1.57264.3` OID root for all of its values.
+
+## Directory
+
+| OID                   | Name                 | Tag Type     | Description                                                                                                                                                        |
+| --------------------- | -------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| 1.3.6.1.4.1.57264.3.1 | TransparencyLogEntry | `BIT STRING` | Proto serialized [TransparencyLogEntry](https://github.com/sigstore/protobuf-specs/blob/4dbf10bc287d76f1bfa68c05a78f3f5add5f56fe/protos/sigstore_rekor.proto#L89). |
```

Comment:
I'd go with utf8string, as it's a more modern type. https://letsencrypt.org/docs/a-warm-welcome-to-asn1-and-der/#bit-string-and-octet-string suggests BIT STRING or OCTET STRING are typically uses for nonces or digests